misc/keycloak-quickstarts-test-helper/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>keycloak-quickstarts-misc-parent</artifactId>
+        <groupId>org.keycloak.quickstarts</groupId>
+        <version>999.0.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>keycloak-quickstarts-test-helper</artifactId>
+    <name>keycloak-quickstarts-test-helper</name>
+    <description>Helper library to test application using Keycloak.</description>
+    <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-client-registration-api</artifactId>
+            <version>${version.keycloak}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-admin-client-tests</artifactId>
```

Comment:
Is it possible to change this to:
```
       <dependency>
            <groupId>org.keycloak</groupId>
            <artifactId>keycloak-admin-client</artifactId>
            <version>${version.keycloak.client}</version>
        </dependency>
```
?

The `keycloak-admin-client-tests` is the admin-client from the `keycloak/keycloak` repository, which is used in the Keycloak testsuite, but we may not want to expose it outside of that. The people should rather use the admin-client from the `keycloak-client` repository in their applications and ideally, that should be also used in quickstarts.

Reply:
Makes sense to me, I've incorporated the changes.